ui/components/multichain/asset-picker-amount/asset-picker-modal/asset-picker-modal-network.tsx

```diff
@@ -104,6 +106,18 @@ export const AssetPickerModalNetwork = ({
     [],
   );
 
+  const [nonTestNetworks, testNetworks] = useMemo(
+    () =>
+      Object.entries(allNetworks).reduce(
+        ([nonTestNetworksList, testNetworksList], [chainId, networkDetail]) => {
+          const isTest = (TEST_CHAINS as string[]).includes(chainId);
+          (isTest ? testNetworksList : nonTestNetworksList).push(networkDetail);
+          return [nonTestNetworksList, testNetworksList];
+        },
+        [[] as NetworkConfiguration[], [] as NetworkConfiguration[]],
+      ),
+    [allNetworks],
+  );
```

Comment:
Using `allNetworks` here adds test networks to the bridge network pickers erroneously. I suggest using `networksList` (which can be a custom list passed in by consuming componetns)

```suggestion
 const [nonTestNetworks, testNetworks] = useMemo(
    () =>
      networksList.reduce(
        ([nonTestNetworksList, testNetworksList], networkDetail) => {
          const isTest = (TEST_CHAINS as string[]).includes(
            networkDetail.chainId,
          );
          (isTest ? testNetworksList : nonTestNetworksList).push(networkDetail);
          return [nonTestNetworksList, testNetworksList];
        },
        [[] as NetworkOption[], [] as NetworkOption[]],
      ),
    [networksList],
  );
```